### DISCUSSION TITLE
모바일에서 그림자(Shadow)가 나오지 않습니다.

### DISCUSSION THREAD

**Discussion by lokimve7** [naverz/zepeto-studio-kor#1136] [5. Q&A]
안녕하세요.
컴퓨터에서 Unity로 실행을 하면 그림자가 정상적으로 나옵니다.
<img width="214" alt="스크린샷 2022-08-07 오후 5 44 13" src="https://user-images.githubusercontent.com/19382832/183282986-9043202f-51c2-4542-9766-1de1469b240b.png">

그러나 QR코드로 모바일에서 실행을 하면 그림자가 나오지 않습니다.

<img width="407" alt="스크린샷 2022-08-07 오후 5 50 45" src="https://user-images.githubusercontent.com/19382832/183283032-e574f21d-c5c1-4762-9cc5-53c223345f1b.png">

모바일에서 그림자가 나오도록 하려면 어떠한 설정을 해줘야 하나요?? 
답변 부탁드립니다.

**Comment by YSHFM** [comment]
저도 최근에 비슷한 문제를 겪은 적이 있었는데요
ProjectSettings > Queality 를 확인하시면 OS별 기본 그래픽 퀄리티 설정이 있습니다.
iOS와 안드로이드 두 환경이 medium으로 되어 있으실텐데 해당 옵션값내 옵션들을 조정해주시면 되겠습니다.

먼저 옵션에 관한 문제가 있을 수 있고 
기본적으로 하이어라키에서 directional light의 인스펙터 값도 확인해보심이 좋으실 것 같습니다.
